routes/patient.js

```diff
@@ -25,56 +22,22 @@ router.post('/patient', (req, res, next) => {
 		}
 
 		//Add a NextOfKin
-		let queryText = `
-			INSERT 
-			INTO HMS-PMS.NextOfKin
-			VALUES (
-			'${req.body.nextOfKinID}',
-			'${req.body.relationship}',
-			'${req.body.address}',
-			'${req.body.phoneNumber}'
-			);
-			`;
-
-		//Add a Patient
-		queryText += `
+		const query =  `
 			INSERT 
 			INTO HMS-PMS.Patient
 			VALUES (
 			'${req.body.patientID}',
-			'${req.body.publicAssurance}',
 			'${req.body.address}',
```

Comment:
Address should be more than 1 value, line (123 something avenue), city, postalCode

Reply:
Just compile all fields and put it in our DB. there is no reason in our MS to divide it up to more than one value for now...

Reply:
Okay, then I will have to save it like this: 123 Something ave, Ottawa, k0k0k0.
And split by comma to update